pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/Dsm.java

```diff
@@ -60,6 +61,9 @@ public class Dsm {
     @SerializedName(ESObjectConstants.ONC_HISTORY_DETAIL_RECORDS)
     List<OncHistoryDetail> oncHistoryDetail;
 
+    @SerializedName(ESObjectConstants.SOMATIC_RESULT_UPLOAD)
```

Comment:
Just curious, what is this file used for?

Reply:
Writing to Elastic.  It's the "thing" that ultimately creates the `dsm` portion of the document in participant_structured.